src/main.rs

```diff
@@ -175,7 +175,7 @@ fn print_changed_files(
 fn should_ignore_current_file(ignored_files: &HashSet<String>, current_file: &Path) -> bool {
     current_file
         .file_name()
-        .and_then(|file_name_os_str| file_name_os_str.to_str())
+        .and_then(std::ffi::OsStr::to_str)
```

Comment:
Matter of personal preference, short form way of calling a closure with one argument